src/main/java/io/openliberty/tools/intellij/lsp4mp/lsp4ij/operations/hover/LSPTextHover.java

```diff
@@ -140,11 +140,18 @@ private CompletableFuture<Integer> getCursorOffset(Editor editor) {
     @Nullable
     @Override
     public String generateDoc(PsiElement element, @Nullable PsiElement originalElement) {
-        Editor editor = LSPIJUtils.editorForElement(element);
+        PsiElement elem = element;
+        Editor editor = LSPIJUtils.editorForElement(elem);
+        if (editor == null) {
```

Comment:
originalElement could be null so you should check it here.

Reply:
Good idea, added null checks for element and originalElement